src/index.js

```diff
@@ -0,0 +1,70 @@
+import './styles.css';
+import {
+  apiGet, sendLike, getLikes, involvementApiGet, involvementApiPost,
+} from './modules/api.js';
+import {
+  generateMovies, generatePopup, generateComments, generateNoComments,
+} from './modules/userInteraction.js';
+import episodesCounter from './modules/episodesCounter.js';
+import commentCounter from './modules/commentsCounter.js';
+
+const mainBody = document.getElementById('main');
+const popup = document.getElementById('popup-window');
+
+window.addEventListener('load', async () => {
+  const numberOfLikes = await getLikes();
+  const epiArray = await apiGet();
+  for (let i = 0; i < epiArray.length; i += 1) {
+    if (epiArray[i].season === 1) {
+      mainBody.innerHTML += generateMovies(epiArray[i], numberOfLikes[i].likes);
```

Comment:
- [ ] It would be better if we can see the like icons as we have in the mockup, our is not showing
![Screenshot from 2023-04-22 22-07-39](https://user-images.githubusercontent.com/46388113/233808286-d7001379-efd9-4a33-8ce0-568993493afa.png)